clesto/__init__.py

```diff
@@ -806,9 +806,201 @@ def __init__(self, data=None, torsion=None):
         # initialize element
         super(Surjection_element, self).__init__(data=data, torsion=torsion)
 
+    def pcompose(self, other, k):
```

Comment:
I don't think we want `pcompose` to be listed a as method of `Surjection_element`. There should only be `compose`. To ways of doing that is to have `pcompose` inside `compose` or to replace `pcompose` with `_pcopose`, since the help function then ignores it.